server/nuxeo-nxr-server/src/main/resources/templates/common-base/client/scripts/taskGC.groovy

```diff
@@ -0,0 +1,33 @@
+import org.apache.logging.log4j.LogManager
+import org.nuxeo.ecm.platform.task.Task
+import org.nuxeo.ecm.core.api.IdRef
+import org.nuxeo.ecm.core.api.ScrollResult
+import org.nuxeo.runtime.transaction.TransactionHelper
+import java.util.HashMap
+
+log = LogManager.getLogger("taskGC.groovy")
+nxql = "SELECT * FROM Document WHERE ecm:mixinType = 'Task'"
+dryRun = true
+session = Context.getCoreSession()
+ScrollResult<String> scrollResult = session.scroll(nxql, 500, 60)
+bucketCount = 0
+documentCount = 0
```

Comment:
Either use it or just remove those variables.

Reply:
Added a removedCount instead of bucketCount
I log at the end:
```log.info("removed {} out of {} tasks", removedCount, documentCount)```

Reply:
Name it `taskCount` maybe so? :p

Reply:
there you go